services/distribution/src/main/java/app/coronawarn/server/services/distribution/config/DistributionServiceConfig.java

```diff
@@ -723,21 +724,67 @@ public void setOverallTimeoutInSeconds(Integer overallTimeoutInSeconds) {
       }
     }
 
+    public static class DeserializedDayPackageMetadata {
+
+      private String region;
+      private String date;
+      private String etag;
+
+      public String getRegion() {
+        return region;
+      }
+
+      public String getDate() {
+        return date;
+      }
+
+      public String getEtag() {
+        return etag;
+      }
+    }
+
+    public static class DeserializedHourPackageMetadata {
```

Comment:
I think this class could extend DeserializedDayPackageMetadata and just add the additional hour field.